recipes/DataProfiler/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "DataProfiler" %}
+{% set version = "0.8.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DataProfiler-{{ version }}.tar.gz
+  sha256: f544b0ab58679ac4c93155439f6fd817241fb480290284a2dde8ce5b6b9e7846
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+    - h5py >=2.10.0
+    - wheel >=0.33.1
+    - future >=0.18.2
+    - numpy >=1.21.1
+    - pandas >=1.1.2
+    - python-dateutil >=2.7.5
+    - pytz >=2020.1
+    - six >=1.15.0
+    - pyarrow >=1.0.1
+    - chardet >=3.0.4
+    - fastavro >=1.0.0
+    - python-snappy >=0.5.4
+    - charset-normalizer >=1.3.6
+    - psutil >=4.0.0
+    - scipy >=1.4.1
+    - requests >=2.27.1,<2.28
+    - networkx >=2.5.1,<2.6
+    - typing-extensions >=3.10.0.2,<3.11
+  run:
+#### requirements.txt
+    - python >=3.7
+    - setuptools
+    - h5py >=2.10.0
+    - wheel >=0.33.1
+    - future >=0.18.2
+    - numpy >=1.21.1
+    - pandas >=1.1.2
+    - python-dateutil >=2.7.5
+    - pytz >=2020.1
+    - six >=1.15.0
+    - pyarrow >=1.0.1
+    - chardet >=3.0.4
+    - fastavro >=1.0.0
+    - python-snappy >=0.5.4
+    - charset-normalizer >=1.3.6
+    - psutil >=4.0.0
+    - scipy >=1.4.1
+    - requests ==2.28.1
+    - networkx ==2.5.1
+    - typing-extensions >=3.10.0.2,<3.11
+#### requirements-reports
+#    - matplotlib-base >=3.3.3,<3.6.1
+#    - seaborn >=0.11.1
+#### requirements-ml.txt
+#    - scikit-learn >=0.23.2  # [not win]
+#    - keras >=2.4.3  # [not win]
+#    - rapidfuzz >=2.6.1  # [not win]
+#    - tensorflow-cpu >=2.6.4  # [not win]
+#    - tqdm >=4.0.0  # [not win]
```

Comment:
These are extras dependencies. If you are not going to ship with those there is no need to add them here as comments.

```suggestion
```